resource/templates/key-value-list.html

```diff
@@ -0,0 +1,35 @@
+<div class="control-group">
+    <div class="key-value-list">
+        <div class="key-value-list__labels flex">
+            <div class="key-value-label__key">{{ keyLabel }}</div>
+            <div class="key-value-label__value">{{ valueLabel }}</div>
+        </div>
+        <div class="key-value-list__elements">
+            <div class="flex key-value-element" ng-repeat="key in keys track by $index">
+                <div custom-dropdown
+                    form="form"
+                    item="keys[$index]"
+                    possible-values="keyOptions"
+                    item-name="{{ keyItemLabel }}"
+                    not-available-values="map"
+                    on-change="dropdownChange"
+                    validity="key-value-list-valid"
+                    class="key-value-element__key"></div>
+                <i class="icon-long-arrow-right"></i>
+                <input type="number"
+                    ng-model="map[key]"
+                    ng-disabled="!key"
+                    min="{{ valueMin }}"
+                    max="{{ valueMax }}"
+                    step="any"
```

Comment:
The step before was 0.01. Do we still want that?

Reply:
Yes indeed we do 😓 Oopsi

Reply:
Actually the step should be a directive param

Reply:
Done in 653a005